spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, :type => :controller do
+  let(:valid_credentials) { ActionController::HttpAuthentication::Basic.encode_credentials("dhh", "secret") }
+  let(:invalid_credentials) { ActionController::HttpAuthentication::Basic.encode_credentials("user", "pass") }
```

Comment:
I think you're using this assignments in different files. So it's a good candidate for shared_context